.gitignore

```diff
@@ -162,3 +162,5 @@ RESTART/
 _dacegraphs
 
 **/testing/output/*
+*.png
+*.nc
```

Comment:
Do we actually want to ignore all png and nc files?

Reply:
I'm okay with leaving it like this, just wanted to be careful when we do actually want to commit a png/nc file.